server/controllers/payroll/reports/payrollReportSocialCharge.handlebars

```diff
@@ -0,0 +1,71 @@
+{{> head title="FORM.LABELS.PAYSLIP"}}
+<body>
+  <!-- body  -->      
+    <header class="row">
+      <div class="col-xs-6" style="font-size: 20px">
+        <h3 style="margin: 0px;"> {{enterprise.name}} </h3>
+        <address style="margin-bottom: 0px;">
+          {{enterprise.location}}
+        </address>
+        <div>{{enterprise.email}}</div>
+        <div>{{enterprise.phone}}</div>
+      </div>
+    </header>    
+    <br>
+    <br>  
+    <h3 class="text-center text-uppercase">
+      <strong> {{ payrollPeriod.label }} </strong> <br>
+      {{translate 'FORM.LABELS.SOCIAL_CHARGES' }}
+    </h3>
+    <br>
+    <br>
+
+  <section>
+    <div class="row">
+      <div class="col-xs-12">
+        <table class="table table-bordered table-condensed table-report table-header-rotated" style="font-size: 10px">
+          <thead>
+            <tr class="text-capitalize">
+              <th style="width: 1%;">{{translate 'TABLE.COLUMNS.NR' }}</th>
+              <th>{{translate 'TABLE.COLUMNS.NAME' }}</th>
+              {{#each rubEnterprises}}
+                <th><div><span >{{ abbr }}</span></div></th>
+              {{/each}}
+              <th style="width: 4%;"><div><span >{{translate 'TABLE.COLUMNS.TOTAL' }}</span></div></th>
+            </tr>
+          </thead>
+
+          <tbody>
+            {{#each dataEmployees}}
+              <tr class="text-capitalize">
+                <td style="text-align: left;"> {{add @index 1}} </td>
```

Comment:
The employees are displayed by alphabetical order, the employees who appear on this report are those who have been selected, the index only allows the order of employees and it can also allow to quickly know the number of employees select